.github/workflows/dependency-review.yml

```diff
@@ -18,4 +18,20 @@ jobs:
         fail-on-severity: high
         allow-licenses: MIT, Apache-2.0, BSD-3-Clause, ISC, BSD-2-Clause, Unlicense, CC0-1.0, 0BSD, X11, MPL-2.0, MPL-1.0, MPL-1.1, MPL-2.0
         fail-on-scopes: development, runtime
-        allow-dependencies-licenses: 'pkg:npm/caniuse-lite'
\ No newline at end of file
+        allow-dependencies-licenses: 'pkg:npm/caniuse-lite'
+     - name: 'Setup Node.js'
+       uses: actions/setup-node@v4
+       with:
+        node-version: '18.x'
+     - name: 'Install dependencies'
+       run: npm install
```

Comment:
Do we need this or the `npm install` command if we are invoking `depcheck` via `npx`? 🤔

Reply:
yeah, I don't think we need. Thanks for catching that. Pushing an update

Reply:
yh, I don't think we need that. Thanks for catching it. Pushing an update